joomla/joomla-cms
# Separate columns for name and link to permission

### Summary of Changes
In Users and Usergroups: Make an extra column for the permissions report. 

### Testing Instructions
See https://github.com/joomla/joomla-cms/pull/22190 

Compare The list views of user groups before and after patch. 
Check if the links call the appropriate permissions. 

### Documentation Changes Required
lyouts have changed a littel bit
